src/components/PostsList/PostsList.jsx

```diff
@@ -1,37 +1,44 @@
-import React from 'react';
+import React, { useState } from 'react';
```

Comment:
```suggestion
import React from 'react';
```
Why do you need useEffect? You don't use it anywhere here